src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = people.filter(men =>
+    century
+      ? Math.ceil(men.died / 100) === century && men.sex === 'm'
+      : (men.sex === 'm')
+  );
+  const menAgeSum = mens.reduce((sumAges, person) =>
```

Comment:
it's not a sum, it's an average
```suggestion
  const menAgeAverage = mens.reduce((sumAges, person) =>
```